reference/configuration/framework.rst

```diff
@@ -4,28 +4,61 @@
 FrameworkBundle Configuration ("framework")
 ===========================================
 
-This reference document is a work in progress. It should be accurate, but
-all options are not yet fully covered.
-
 The FrameworkBundle contains most of the "base" framework functionality
-and can be configured under the ``framework`` key in your application configuration.
+and can be configured under the ``framework`` key in your application
+configuration. When using XML, you can use the
+``http://symfony.com/schema/dic/symfony`` namespace.
+
 This includes settings related to sessions, translation, forms, validation,
 routing and more.
 
+.. tip::
+
+   The XSD schema is available at
+   ``http://symfony.com/schema/dic/symfony/symfony-1.0.xsd``.
```

Comment:
I'm not sure about the uploaded file, but this is meant as a reference for when people want to use XML schema validation in Symfony (the local file is used for xsv in Symfony). That's also why it's a literal and not a link.